great-idea/services.html

```diff
@@ -7,7 +7,7 @@
   <title>Great Idea!</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/services.css">
```

Comment:
Great job knowing to change the linked CSS sheet, since you created a new services CSS file.